web/src/components/namespace/project-cards/project-card.tsx

```diff
@@ -55,19 +58,17 @@ export const ProjectCard: FC<Props> = ({ project }) => {
             </span>
           ) : null}
         </div>
-        { !isLoading ? 
+        { !isLoading && (
           <ProjectCardDropdown
```

Comment:
Right here:
 `key={isStarred}` and that will fix your `isLocalStarred` problem with `useEffect` (I think)

Reply:
Fixed